test/e2e/framework/network_pods.go

```diff
@@ -185,11 +200,12 @@ func (np *NetworkPod) buildTCPCheckConnectorPod() {
 					Image: "busybox",
 					// We send the string 50 times to put more pressure on the TCP connection and avoid limited
 					// resource environments from not sending at least some data before timeout.
-					Command: []string{"sh", "-c", "for in in $(seq 50); do echo connector says $SEND_STRING; done | nc -v $REMOTE_IP $REMOTE_PORT -w 8 >/dev/termination-log 2>&1"},
+					Command: []string{"sh", "-c", "for in in $(seq 50); do echo [dataplane] connector says $SEND_STRING; done | for i in $(seq $CONN_TRIES); do if nc -v $REMOTE_IP $REMOTE_PORT -w 2; then break; fi; done >/dev/termination-log 2>&1"},
```

Comment:
Please keep the -w to "8", otherwise in travis, sometimes it's slow enough to fail with "2" seconds. I guess it will make the negative tests slower, but... it will avoid random failures.

Reply:
I added retries for ncat  so I thought it would be OK to reduce -w to 8 but I can leave it at 8 and reduce the retries accordingly.

Reply:
I compromised and set -w to 4 with 10 retries by default. I think that should suffice.